src/server/rpc/procedures/roboscape/roboscape.js

```diff
@@ -52,6 +52,7 @@ var RoboScape = function () {
 RoboScape.serviceName = 'RoboScape';
 RoboScape.prototype._robots = {};
 
+// fetch the robot, create one if necessary
 RoboScape.prototype._addRobot = function (mac_addr, ip4_addr, ip4_port) {
```

Comment:
I thought we were renaming this function?

Reply:
good catch `getRobot` `getOrCreateRobot` ? any other suggestions

Reply:
renamed to `getOrCreateRobot`